en/docs/developer-portal/applications/samples/android.md

```diff
@@ -10,10 +10,15 @@ using the [PKCE extension](https://tools.ietf.org/html/rfc7636)
 
 1. Download the [android sample](https://github.com/wso2-extensions/identity-samples-android/archive/master.zip) and
  Unzip it.
+ 
+2. Open the sample in Android Studio IDE.
 
 
 ### Configuration
 
+Need to add application configuration in your android project in order to authenticate with Identity server using
```

Comment:
You need to add

with WSO2 Identity Server (Please replace identity server with WSO2 Identity Server everywhere)